### PR TITLE
feat:Rename class names and types to PascalCase

### LINKED ISSUE #469: Rename class names and types to PascalCase
### Objective
We want to rename all the class names, typescript types and enums used in the project to PascalCase

### Acceptance Criteria
- Rename all class names, typescript types and enums used in the project to PascalCase
- Don't rename anything else. Functions, variables, constants and file names are out of scope for this ticket
- Make sure that the test run and nothing breaks

### PR DESCRIPTION
**Pull request by divinecharlotte**
### Renaming class names and types to PascalCase
We wanted to rename all the class names, typescript types and enums used in the project to PascalCase

### What I did

- Renamed all class names, typescript types, and enums used in the project to PascalCase
- Didn't rename anything else. Functions, variables, constants, and file names are out of scope for this ticket
- Made sure that the test run and nothing breaks

### Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] No existing features have been broken without good reason.
- [x] The code style guideline have been followed
- [x] Documentation has been updated to reflect your changes.

### CODE REVIEW

File: apps/drec-api/src/pods/certificate-log/certificate-log.service.ts [hunk 1]
```diff
@@ -581,7 +581,7 @@ export class CertificateLogService {
   ): Promise<{
     certificatelog:
       | CertificateNewWithPerDeviceLog[]
-      | CertificateWithPerdevicelog[];
+      | CertificateWithPerDevicelog[];
```
Comment (Kingcedru) [inline apps/drec-api/src/pods/certificate-log/certificate-log.service.ts]: `log` is not starting with a capital letter

**Review by Kingcedru — CHANGES_REQUESTED** [review]

**Review by Kingcedru — DISMISSED** [review]
Good

File: apps/drec-api/src/pods/sdgbenefit/sdgbenefit.module.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { Module } from '@nestjs/common';
...
-import { SdgBenefit } from './sdgbenefit.entity';
+import { SDGBenefit } from './sdgbenefit.entity';
 @Module({
-  imports: [TypeOrmModule.forFeature([SdgBenefit])],
+  imports: [TypeOrmModule.forFeature([SDGBenefit])],
   controllers: [SdgbenefitController],
   providers: [SdgbenefitService],
 })
-export class SdgbenefitModule {}
+export class SDGbenefitModule {}
```
Comment (musayann) [inline apps/drec-api/src/pods/sdgbenefit/sdgbenefit.module.ts]: This should be `SDGBenefitModule`

**Review by musayann — CHANGES_REQUESTED** [review]

File: apps/drec-api/src/drec.module.ts [hunk 3]
```diff
@@ -48,9 +48,9 @@ import { AggregateMeterRead } from './pods/reads/aggregate_readvalue.entity';
 import { HistoryIntermediate_MeterRead } from './pods/reads/history_intermideate_meterread.entity';
 import { CheckCertificateIssueDateLogForDeviceEntity } from './pods/device/check_certificate_issue_date_log_for_device.entity';
 import { CheckCertificateIssueDateLogForDeviceGroupEntity } from './pods/device-group/check_certificate_issue_date_log_for_device_group.entity';
-import { CountrycodeModule } from './pods/countrycode/countrycode.module';
-import { SdgbenefitModule } from './pods/sdgbenefit/sdgbenefit.module';
-import { SdgBenefit } from './pods/sdgbenefit/sdgbenefit.entity';
+import { CountryCodeModule } from './pods/countrycode/countrycode.module';
+import { SDGbenefitModule } from './pods/sdgbenefit/sdgbenefit.module';
```
Comment (musayann) [inline apps/drec-api/src/drec.module.ts]: This should also be `SDGBenefitModule`

**Review by musayann — CHANGES_REQUESTED** [review]

**Review by musayann — APPROVED** [review]

**Pull request merged**